extra/1-radio-stations.js

```diff
@@ -25,7 +33,17 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
-
+function getStations() {
+  let allFrequency = getAllFrequencies();
+  let availableFrequency = [];
+  for(let i = 0; i < allFrequency.length; i++) {
+    let frequencies = allFrequency[i];
+    if(isRadioStation(frequencies)) {
+      availableFrequency.push(frequencies);
+    }
+  }
+  return availableFrequency;
+}
```

Comment:
Well done Abiyaa good work. I'm impressed the way you did the extra task

Reply:
Thanks, Haseeb. I got some useful suggestions from HWC.